astroplan/observer.py

```diff
@@ -1958,3 +1958,27 @@ def tonight(self, time=None, horizon=0 * u.degree, obswl=None):
         end_time = self.sun_rise_time(start_time, which='next', horizon=horizon)
 
         return start_time, end_time
+
+    @property
+    def longitude(self):
+        """longitude : float, str, `~astropy.units.Quantity` (optional)
+            The longitude of the observing location. Should be valid input for
+            initializing a `~astropy.coordinates.Longitude` object.
+        """
```

Comment:
The docstrings here should be changed. These are describing params as if the property could be set, but it cannot (which might wanted to be added, @bmorris3?).

Can you change the docstrings to match a getter property?

Reply:
Sure, I actually just copied it from the Observer class's own docstring. Is there a preferred style for getter property docstrings? (Otherwise I can try to find an example from the codebase).

Reply:
Ok I tried something, but let me know if you have any suggestions on it.